internal/tui/components/app/app.go

```diff
@@ -423,9 +473,11 @@ func (m *Model) handleKeyMsg(msg tea.KeyMsg) tea.Cmd {
 
 		case key.Matches(msg, keymap.KeyMap.Reload):
 			if m.currentPage.DoesReload() {
+				m.getCurrentPageModel().HideToast()
```

Comment:
I actually don't think we want this - people might manually be refreshing the page every second or so manually with `r` and hide the toast too quickly